Cython/Shadow.py

```diff
@@ -506,6 +518,9 @@ def _specialized_from_args(signatures, args, kwargs):
 
 type_ordering = [py_int, py_long, py_float, py_complex]
 
+PyTypeObject = type
+PyObject_TypeCheck = isinstance
+
```

Comment:
Why are they here? I don't think we should expose them to Python code.

Reply:
They're defined in

https://github.com/cython/cython/blob/98e5b9b0229bf2b9cad8e485dcebb2f9c0eacee6/Cython/Compiler/CythonScope.py#L84-L100

although that looks very much like an implementation detail to me. So I think pure-Python mode users could access them, but I'm not sure why they'd want to.

I'll manually exclude them for now, and leave investigating why they're there for the future